Userland/Games/Flood/MainWidget.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGUI/Widget.h>
+
+namespace Flood {
+
+class MainWidget : public GUI::Widget {
+    C_OBJECT(MainWidget)
```

Comment:
This can be C_OBJECT_ABSTRACT, since the auto-generated constructor functions shouldn't be used.